tests/unittests/Foundation/NSComparisonPredicateTests.m

```diff
@@ -630,7 +630,7 @@ - (void)dealloc {
 
     ASSERT_TRUE_MSG(comparisonPredicate != nil, "FAILED: comparisonPredicate should be non-null!");
 
-    ASSERT_TRUE_MSG([comparisonPredicate evaluateWithObject:nil substitutionVariables:nil], "FAILED: Comparison failed.");
+    ASSERT_TRUE_MSG(![comparisonPredicate evaluateWithObject:nil substitutionVariables:nil], "FAILED: Comparison failed.");
```

Comment:
this had to do with an issue with capitalization option.  we didn't support the full fledged options. now we do.